microsoftgraph/microsoft-graph-explorer-v4
# Fix: Autocomplete filter

## Overview
Fixes #2431  

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes


## Testing Instructions

* How to test this PR
Type a query like /messages?$filter
Notice that all filters added can be typed without issues
